doc/dox_comments/header_files/ssl.h

```diff
@@ -568,6 +568,26 @@ WOLFSSL_API int wolfSSL_use_old_poly(WOLFSSL*, int);
 WOLFSSL_API int wolfSSL_dtls_import(WOLFSSL* ssl, unsigned char* buf,
                                                                unsigned int sz);
 
+
+/*!
+    \brief Used to import a serialized TLS session. In most cases
+    wolfSSL_get_session should be used instead, this function is for exporting
```

Comment:
This text is confusing because it describes `wolfSSL_get_session` for exporting on an import function. Could this be clarified, why `wolfSSL_tls_import` vs `wolfSSL_get_session` would be used? Same with `wolfSSL_tls_export` doc below.